git_machete/client.py

```diff
@@ -1968,10 +1965,9 @@ def checkout_github_prs(self,
         debug('checkout_github_pr()',
```

Comment:
While you're here... pls update the method name in `debug` (unless there some Python magic to like `__current_method_name__`?)

Reply:
yeah, its possible